### PR TITLE
Update screenshots and steps for new UI architecture

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1216

I didn't replace all screenshots, as some were either unused in docs or originally made because our UI used to be less intuitive. 

Release notes for this change will come in a separate PR

### CODE REVIEW

File: astro/api-keys.md [hunk 1]
```diff
@@ -25,11 +25,14 @@ When using Deployment API keys, keep in mind the following:
...
-2. In the **API keys** menu, click **Add API Key**:
+
+2. Click **API Keys**.
+
+3. Click **+ API Key**:
 
     ![Add API key button](/img/docs/add-api-key.png)
 
-3. Give the key a name and description, then click **Create API Key**:
+4. Give the key a name and description, then click **Create API Key**:
```
Comment (kwcanuck) [inline astro/api-keys.md:35]:
```suggestion
4. Enter a name and a description for the API key and then click **Create API Key**:
```

File: astro/api-keys.md [hunk 2]
```diff
@@ -66,13 +69,14 @@ After setting the variables, running `astro deployment update` works for the Dep
 ## Delete an API key
 
 1. In the Cloud UI, select a Workspace and then select a Deployment.
-2. In the menu for the API key you want to delete, click **Edit**:
 
-    ![Edit API key button](/img/docs/edit-api-key.png)
+2. Click **API Keys**
 
-3. Click **Delete API Key**, and then follow the onscreen prompt to finalize the deletion:
+3. Click the **Edit** button next to your API key.
```
Comment (kwcanuck) [inline astro/api-keys.md:75]:
```suggestion
3. Click **Edit** next to your API key.
```

image

File: astro/api-keys.md [hunk 3]
```diff
@@ -66,13 +69,14 @@ After setting the variables, running `astro deployment update` works for the Dep
...
+
+    ![Edit API key button](/img/docs/edit-api-key.png)
 
-    ![Delete API key button](/img/docs/delete-api-key.png)
+3. Click **Delete API Key**, and then follow the onscreen prompt to finalize the deletion.
```
Comment (kwcanuck) [inline astro/api-keys.md:79]:
```suggestion
3. Click **Delete API Key**, enter `Delete`, and then click **Yes, Continue**.
```

File: astro/api-keys.md [hunk 4]
```diff
@@ -66,13 +69,14 @@ After setting the variables, running `astro deployment update` works for the Dep
...
 
 If you delete an API key, make sure that no existing CI/CD pipelines are using it. Once deleted, an API key and secret cannot be recovered. If you unintentionally delete an API key, create a new one and update any CI/CD workflows that used the deleted API key.
```
Comment (kwcanuck) [inline astro/api-keys.md:81]:
```suggestion
If you delete an API key, make sure that no existing CI/CD pipelines are using it. Once deleted, an API key and secret cannot be recovered. If you unintentionally delete an API key, create a new one and update any CI/CD workflows that used the deleted API key.
```

This should follow the heading. It's important information users should have BEFORE completing the procedure.

File: astro/configure-deployment-resources.md [hunk 5]
```diff
@@ -106,7 +106,7 @@ These calculations are computed by KEDA every 10 seconds. For more information o
 
 ## Scheduler resources
 
-The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks once dependencies have been met. By adjusting the **Scheduler Count** slider in the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Scheduler Resources**.
+The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks once dependencies have been met. By adjusting the **Scheduler Count** slider in the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**.
```
Comment (kwcanuck) [inline astro/configure-deployment-resources.md:109]:
```suggestion
The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks when the dependencies are met. By adjusting the **Scheduler Count** slider in the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**.
```

File: astro/configure-deployment-resources.md [hunk 6]
```diff
@@ -120,11 +120,8 @@ If you haven't created a Deployment, see [Create a Deployment](create-deployment
 2. Select a Deployment.
 3. Based on the resource setting you want to change, click one of the following buttons:
 
-    ![Location of all Deployment resource setting buttons](/img/docs/deployment-settings.png)
-
-    - To create a new worker queue, click **Add Worker Queue**.
-    - To change an existing worker queue, click the **Edit** button next to that worker queue's entry in the table in the **Worker Queues** section.
-    - To change scheduler resources, click the **Edit** button in the **Scheduler Settings** section.
+    - To create a worker queue or change an existing worker queue, click **Worker Queues**.
```
Comment (kwcanuck) [inline astro/configure-deployment-resources.md:123]:
```suggestion
    - To change the settings of an existing worker queue, click **Edit*, adjust the settings, and then click **Update Queue**.
```

File: astro/configure-deployment-resources.md [hunk 7]
```diff
@@ -120,11 +120,8 @@ If you haven't created a Deployment, see [Create a Deployment](create-deployment
 2. Select a Deployment.
 3. Based on the resource setting you want to change, click one of the following buttons:
```
Comment (kwcanuck) [inline astro/configure-deployment-resources.md:121]:
```suggestion
3. Select one of the following options to adjust the settings of a Worker Queue or a Scheduler:
```

File: astro/configure-deployment-resources.md [hunk 8]
```diff
@@ -120,11 +120,8 @@ If you haven't created a Deployment, see [Create a Deployment](create-deployment
...
 
-    ![Location of all Deployment resource setting buttons](/img/docs/deployment-settings.png)
-
-    - To create a new worker queue, click **Add Worker Queue**.
-    - To change an existing worker queue, click the **Edit** button next to that worker queue's entry in the table in the **Worker Queues** section.
-    - To change scheduler resources, click the **Edit** button in the **Scheduler Settings** section.
+    - To create a worker queue or change an existing worker queue, click **Worker Queues**.
+    - To change scheduler resources, click **Configuration** and then click **Edit Configuration**.
```
Comment (kwcanuck) [inline astro/configure-deployment-resources.md:124]:
```suggestion
    - To change the settings of an existing Scheduler, click **Edit** in the **Scheduler Settings** area,  adjust the settings, and then click **Update Scheduler**.
```

File: astro/create-deployment.md [hunk 9]
```diff
@@ -22,8 +22,11 @@ If you prefer, you can also run the `astro deployment create` command in the Ast
 :::
 
 1. Log in to the [Cloud UI](https://cloud.astronomer.io) and select a Workspace.
-2. Click the **Deployment** button.
+
+2. In the **Deployments** page, click the **+ Deployment** button.
```
Comment (kwcanuck) [inline astro/create-deployment.md:26]:
```suggestion
2. On the **Deployments** page, click **Deployment**.
```
image

You _might_ want to request that hover text be added to the Deployments button.

File: astro/deployment-metrics.md [hunk 10]
```diff
@@ -104,7 +96,7 @@ Deployment health can have one of two statuses:
     - Your Deployment was recently created and the Airflow webserver and scheduler are still spinning up.
     - Your Deployment's webserver and/or scheduler are restarting or otherwise not in a healthy, running state.
 
-If your Deployment is unhealthy, we recommend checking the status of your tasks and waiting for a few minutes. If your Deployment is unhealthy for more than 5 minutes, we recommend [reviewing scheduler logs](scheduler-logs.md) in the Cloud UI or reaching out to [Astronomer support](https://cloud.astronomer.io/support).
+If your Deployment is unhealthy, we recommend checking the status of your tasks and waiting for a few minutes. If your Deployment is unhealthy for more than 5 minutes, we recommend [reviewing scheduler logs](view-logs.md#view-airflow-scheduler-logs) in the Cloud UI or reaching out to [Astronomer support](https://cloud.astronomer.io/support).
```
Comment (kwcanuck) [inline astro/deployment-metrics.md:99]:
```suggestion
If your Deployment is unhealthy, Astronomer recommends checking the status of your tasks and waiting for a few minutes. If your Deployment is unhealthy for more than 5 minutes, [review the scheduler logs](view-logs.md#view-airflow-scheduler-logs) in the Cloud UI or contact [Astronomer support](https://cloud.astronomer.io/support).
```

File: astro/environment-variables.md [hunk 11]
```diff
@@ -26,26 +26,36 @@ If you prefer to work with the Astro CLI, you can create and update environment
...
 
 ### Edit existing values
 
 After you set an environment variable key, only the environment variable value can be modified. You can modify environment variables that are set as secret. However, the variable value is never shown. When you modify a secret environment variable, you'll be prompted to enter a new value.
 
 1. In the Cloud UI, select a Workspace and then select a Deployment.
-2. Click **Edit Variables**.
-3. Click **Edit value** next to the value you want to edit.
+
+2. Click **Variables**
```
Comment (kwcanuck) [inline astro/environment-variables.md:46]:
```suggestion
2. Click **Environment Variables**
```

File: astro/environment-variables.md [hunk 12]
```diff
@@ -26,26 +26,36 @@ If you prefer to work with the Astro CLI, you can create and update environment
...
+
+3. Click **Edit Variables**.
+
+4. Click the pencil icon next to the value you want to edit.
```
Comment (kwcanuck) [inline astro/environment-variables.md:50]:
```suggestion
4. Click **Edit value** next to the value you want to edit.
```
image

image

Most users will know that a pencil icon is associated with editing.

File: astro/view-logs.md [hunk 13]
```diff
@@ -42,31 +42,27 @@ Logs for the Airflow webserver, worker, and triggerer are not available for Depl
...
+You can access the past 24 hours of scheduler logs for any Deployment on its **Logs** page. Logs are color-coded according to their type. Scheduler logs can help you understand scheduler performance and indicate if a task failed due to an issue with the scheduler. For more information on configuring the scheduler on Astro, see [Scheduler resources](configure-deployment-resources.md#scheduler-resources).
 
-When a Deployment generates more than 500 lines of logs in 24 hours, only the most recent 500 lines are shown. If there are no scheduler logs available for a given Deployment, the following message appears:
-
-```
-No matching events have been recorded in the past 24 hours.
-```
-
-Typically, this indicates that the Deployment you selected does not currently have any DAGs running.
+1. In the Cloud UI, select a Workspace and Deployment.
```
Comment (kwcanuck) [inline astro/view-logs.md:47]:
```suggestion
1. In the Cloud UI, select a Workspace and then a Deployment.
```

**Review by kwcanuck — APPROVED** [review]

File: astro/api-keys.md [hunk 14]
```diff
@@ -66,13 +69,14 @@ After setting the variables, running `astro deployment update` works for the Dep
 ## Delete an API key
```
Comment (jwitz) [inline astro/api-keys.md:70]:
```suggestion

If you delete an API key, make sure that no existing CI/CD pipelines are using it. Once deleted, an API key and secret cannot be recovered. If you unintentionally delete an API key, create a new one and update any CI/CD workflows that used the deleted API key.

```

File: astro/api-keys.md (hunk 4 above)
Comment (jwitz) [inline astro/api-keys.md:81]:
```suggestion
```

File: astro/environment-variables.md (hunk 11 above)
Comment (jwitz) [reply astro/environment-variables.md:46]: The button says **Variables** only, oddly Screen Shot 2022-09-21 at 3 57 12 PM

File: astro/view-logs.md [hunk 15]
```diff
@@ -42,31 +42,27 @@ Logs for the Airflow webserver, worker, and triggerer are not available for Depl
...
 
-1. In the Cloud UI, select a Workspace.
-2. Click **Logs** on the left menu.
+2. Click **Logs**.
 
-    ![Logs icon and button](/img/docs/log-location.png)
-
-3. Select a Deployment in the **Select a Deployment** menu.
-4. Optional. Select one or more options in the **Log Level** menu and click **Apply**. These are the available options:
+3. Optional. Select one or more options in the **Log Level** menu and click **Apply**. These are the available options:
```
[the remark this answers is not in the document]
Comment (jwitz) [reply astro/view-logs.md:51]: Not for this new update. You have already selected a Deployment in step 1

File: astro/environment-variables.md (hunk 12 above)
Comment (jwitz) [reply astro/environment-variables.md]: Issue created: https://github.com/astronomer/astro/issues/4037

File: astro/api-keys.md [hunk 16]
```diff
@@ -25,11 +25,14 @@ When using Deployment API keys, keep in mind the following:
 To create an API key for a Deployment:
```
Comment (paolaperaza) [inline astro/api-keys.md:25]:
```suggestion
```

I suppose delete this, and some of the space around it?

File: astro/configure-deployment-resources.md (hunk 5 above)
Comment (paolaperaza) [inline astro/configure-deployment-resources.md:109]:
```suggestion
The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks once dependencies have been met. By adjusting the **Scheduler Count** slider in the **Configuration** tab of the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**.
```

File: astro/configure-deployment-resources.md [hunk 17]
```diff
@@ -118,13 +118,10 @@ If you haven't created a Deployment, see [Create a Deployment](create-deployment
...
-    ![Location of all Deployment resource setting buttons](/img/docs/deployment-settings.png)
-
-    - To create a new worker queue, click **Add Worker Queue**.
-    - To change an existing worker queue, click the **Edit** button next to that worker queue's entry in the table in the **Worker Queues** section.
-    - To change scheduler resources, click the **Edit** button in the **Scheduler Settings** section.
+    - To change the settings of an existing worker queue, click **Edit** in the **Worker Queues** area, adjust the settings, and then click **Update Queue**.
+    - To change scheduler resources, click **Configuration** and then click **Edit Configuration**.
 
 4. Edit the Deployment resource settings. For more information about these settings, review the content in this topic.
 5. Click **Update Deployment**.
```
Comment (paolaperaza) [inline astro/configure-deployment-resources.md:127]: Doesn't look like there's an `**Update Deployment** button anymore. Should we just split these sections up into editing scheduler vs. worker queue?

For worker queue, it's **Update queue* and for scheduler it's just **Update**

File: astro/deployment-metrics.md [hunk 18]
```diff
@@ -11,15 +11,7 @@ The Cloud UI exposes a suite of observability metrics that show real-time data r
 
 Located in the Workspace view of the Cloud UI, the **Analytics** page contains a suite of metrics for your Deployments. This page includes metrics that give you insight into the performance of both your data pipelines and infrastructure. Because metrics are collected in real time, you can use this page to detect irregularities in your pipelines or infrastructure as they happen.
```
Comment (paolaperaza) [inline astro/deployment-metrics.md:12]: Don't think this is the case? There is no **Analytics** page in the Workspace view anymore, unless you're talking about the main list of Deployments view?

**Review by paolaperaza — COMMENTED** [review]
Couple comments here @jwitz ! Give these a look and I'll send you back a ✅ later today?

File: astro/api-keys.md (hunk 16 above)
Comment (jwitz) [reply astro/api-keys.md:25]: I think Rob has this covered in a separate PR https://github.com/astronomer/docs/pull/1230

File: astro/configure-deployment-resources.md (hunk 17 above)
Comment (jwitz) [inline astro/configure-deployment-resources.md:127]:
```suggestion
5. Click **Update**.
```

**Comment by jwitz** [comment]
@paolaperaza I should have resolved all comments - feel free to merge this if it looks good and I'm AFK

**Pull request merged**